platform/ui/src/elements/Icon/icons/angle-open.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="578" height="763" viewBox="0 0 578 763" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Where is the source of this SVG? Is it allowed to be re-used without attribution?

Reply:
It's a SVG designed by @dbejarano32 for OHIF project and any fork of this project!